### DISCUSSION TITLE
Buttler auth are not available to download

### DISCUSSION THREAD

**Discussion by marcosengineer** [ptarmiganlabs/butler-auth#37] [General]
Hi everyone

Buttler auth are not available to download i need the solution to implements in a customer

Can someone helpme please?

**Comment by mountaindude** [comment]
Hi Marcos,

The release page shows what's available for download: https://github.com/ptarmiganlabs/butler-auth/releases

Should be mentioned that I've not put a lot of focus on this project during the past year. 
The tool worked really well a year ago and still works well in places where I know it's being used, so no fundamental problems really.
But: Given the lack of updates during the past year, the tool does not have the most recent versions of submodules etc. Which means the latest bug fixes, security patches etc are not included in the latest Butler Auth version.

What would be needed is someone doing an update of all dependencies in the project, possibly also package things into stand-alone binaries for easier installation. And wrap the whole thing up in a new release.

I don't have the time for this though - but hey, this is open source! Anyone can fork the repository and make contributions via pull requests!

Best regards,
Göran
